libsrc/Makefile

```diff
@@ -91,7 +93,7 @@ define INSTALL_recipe
 
 $(if $(prefix),,$(error variable `prefix' must be set))
```

Comment:
But the problem is that putting aside the check, it breaks when one supplies the DESTDIR.